Public/Src/Engine/Dll/EngineSchedule.cs

```diff
@@ -944,6 +954,15 @@ void AddToNonScrubbableAbsolutePaths(AbsolutePath[] paths)
                 return false;
             }
 
+            // The filter may or may not have already been computed depending on whether there was a graph hit or not.
+            if (filter == null && !TryGetPipFilter(loggingContext, Context, commandLineConfiguration, configuration, out filter))

```

Comment:
Just moved this up because later on the filter might be needed before going through sideband files.  Reason: we want to know the filtered graph (i.e., the actual requested pips) before deciding if we have all corresponding sideband files in place.